README.md

```diff
@@ -135,6 +135,27 @@ Similarly, in the container started on `$HOST2`...
 So there we have it, two containers on separate hosts happily talking
 to each other.
 
+## OSX
+
+For usage on OSX (e.g. with Docker Machine and/or Boot2Docker) we need
+to make sure that the VM is running and configured before launching
+weave. For example, for a VM named "weave-1" on docker-machine:
+
+    osx$ docker-machine create -d virtualbox weave-1
+    osx$ docker-machine start weave-1
+    osx$ eval "$(docker-machine env weave-1)"
+
```

Comment:
These instructions should a) go into the Installation section, and b) finish right here (the remainder is unnecessary/duplicate info).

And we need a separate branch for this change, since we must not update the master README with instructions that only work for the next release. Look back at how we handled this for the README changes that were part of the 1.1 release.